EIPS/eip-2502.md

```diff
@@ -0,0 +1,462 @@
+
+```plantuml
+@startuml
+rectangle Root {
+    :ColdWallet: -> :HotWallet: : delegate
+}
+
+rectangle Topic1 {
+    :HotWallet: -> :Specialist1: : delegate
+}
+
+rectangle Topic2 {
+    :HotWallet: -> :Specialist2: : delegate
+}
+@enduml
+```
+
+#### Sub Delegations (Micro Topics)
+
+Sub Delegations operate under their own rules, as having a Default Delegate, however would be common for them following the same rules as root delegation. 
+
+When an address don't have a delegate set, it looks up in the parent delegation. If the parent delegation returns zero, the default delegate is looked up at the sub delegation. The address zero delegate (default delegate) is only looked up in the sub delegation.
+
+##### Default Delegate
+
+When a user didn't set a delegate in a parent delegation, it will lookup to the defaultDelegate, which in root topic is "undefined", but in sub topics a default delegate can be defined.  
+
+The default delegate is a option sub delegations can implement to facilitate developers of building a DAO, where all stakeholders by default would delegate to the developer address. Some democracies may allow change of default delegate, others not. 
+
+Technically, Default Delegate is the delegate of the address zero, which appears when a delegate was not set. At root delegation, the technical end of linear delegations is the address zero, which ends up in a circular delegation with itself. 
+
+
+### Delegation Interface
+
+```solidity
+pragma solidity >=0.5.0 <0.7.0;
+
+/**
+ * @author Ricardo Guilherme Schmidt (Status Research & Development GmbH)
+ */
+interface Delegation {
+
+    event Delegate(address who, address to);
+
+    /**
+     * @notice Set `msg.sender` delegate.
+     * `_to == address(0)` resets to parent delegation
+     * `_to == msg.sender` ends delegate chain.
+     * @param _to Address the caller will delegate to.
+     */
+    function delegate(address _to) external;
+
+    /**
+     * @notice Reads `_who` configured delegation in this level or
+     * from parent level if `_who` never defined/reset.
+     * @param _who What address to lookup.
+     * @return The address `_who` chosen delegate to.
+     */
+    function delegatedTo(address _who) external view returns (address);
+
+    /**
+     * @notice Reads `_who` configured delegation at block number `_block` or from parent level if `_who` never defined/reset.
+     * @param _who What address to lookup.
+     * @param _block Block number of what height in history.
+     * @return The address `_who` chosen delegate to.
+     */
+    function delegatedToAt(address _who, uint _block) external view returns (address);
+}
+```
+
+### Deployment 
+
+#### Root Topic
+
+The root delegation address, for any ethereum chain, is `0x2502205225022052250220522502205225022052` which is deployed by the Singleton Factory [ERC-2470] using the salt `0x2502205225022052250220522502205225022052250220522502` and using the bytecode provided below.
```

Comment:
Why is the deployment salted, this seems like it just complicates the deployment process and doesn't provide any value?  (include answer in Rationale)